json-formatter/CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ This document is formatted according to the principles of [Keep A CHANGELOG](htt
 
 ### Changed
 
+* Handle the `JavaMethod` and `JavaStackTraceElement` fields in `SourceReference`:
+  - outputs `<className>.<methodName>` for `JavaMethod`
```

Comment:
Should be `<className>.<methodName>(<parameter1>,<parameter2>, ect)`

Reply:
I've fixed this @mpkorstanje